OpenSim/Sandbox/MatlabScripts/createActuatorsFile.m

```diff
@@ -0,0 +1,156 @@
+% and supported by the US National Institutes of Health (U54 GM072970,    %
+% R24 HD065690) and by DARPA through the Warrior Web program.             %
+%                                                                         %
+% Copyright (c) 2005-2016 Stanford University and the Authors             %
+% Author(s): James Dunne                                                  %
+%                                                                         %
+% Licensed under the Apache License, Version 2.0 (the "License");         %
+% you may not use this file except in compliance with the License.        %
+% You may obtain a copy of the License at                                 %
+% http://www.apache.org/licenses/LICENSE-2.0.                             %
+%                                                                         %
+% Unless required by applicable law or agreed to in writing, software     %
+% distributed under the License is distributed on an "AS IS" BASIS,       %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+% implied. See the License for the specific language governing            %
+% permissions and limitations under the License.                          %
+% ----------------------------------------------------------------------- %
+
+% Author: James Dunne, Chris Dembia, Tom Uchida,
+% Ajay Seth, Ayman Habib, Shrinidhi K. Lakshmikanth, Jen Hicks.
+
+% Import OpenSim Libraries
+import org.opensim.modeling.*
+
+% open dialog boxes to select the model
+[filename, pathname] = uigetfile('*.osim', 'Select an OpenSim Model File');
+
+% get the model path
+modelFilePath = fullfile(pathname,filename);
+
+% Generate an instance of the model
+model = Model(modelFilePath);
+
+% Get the number of coordinates  and a handle to the coordainte set
+nCoord = model.getCoordinateSet.getSize();
+coordSet = model.getCoordinateSet();
+
+% Create some empty vec3's for later.
+massCenter = Vec3();
+axisValues = Vec3();
+
+% Create an empty Force set
+forceSet = ForceSet();
+
+% Create the underlying computational System and return a handle to the State
+state = model.initSystem();
+
+optimalForce = 1;
+
+%% Start going through the coordinates, creating an actuator for each
+for iCoord = 0 : nCoord - 1
+
+    % get a reference to the current coordinate
+    coordinate = coordSet.get(iCoord);
+    % If the coodinate is constrained, don't add an actuator
+    if coordinate.isConstrained(state)
+        continue
+    end
+
+    % get the joint, parent and child names for the coordiante
+    joint = coordinate.getJoint();
+    parentName = joint.getParentFrame().getName();
+    childName = joint.getChildFrame().getName();
+
+    % If the coordinates parent body is connected to ground, we need to
+    % add residual actuators (torque or point).
+    if strmatch(parentName, model.getGround.getName() )
+
+        % if the joint type is custom or free
```

Comment:
This comment is not useful. Perhaps provide a short explanation of what the code block does, or just delete the comment.